paf-mvp-audit/src/main.ts

```diff
@@ -1,7 +1,10 @@
 import { Locale } from './locale';
 import { Controller } from './controller';
-import { Log } from '@core/log';
 import { Window } from '@frontend/global';
+import { Log, LogLevel } from '@core/log';
+
+// Debug level while playing with MVP
+Log.level = LogLevel.Debug;
```

Comment:
⚠️ not directly related to this PR, but should be done because now the audit log is not logging anything